test/e2e/fishapp/shared/manifest_builder.go

```diff
@@ -0,0 +1,275 @@
+package shared
+
+import (
+	"fmt"
+	appmeshv1beta1 "github.com/aws/aws-app-mesh-controller-for-k8s/pkg/apis/appmesh/v1beta1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	defaultAppImage       = "970805265562.dkr.ecr.us-west-2.amazonaws.com/colorteller:latest"
+	defaultHTTPProxyImage = "abhinavsingh/proxy.py:latest"
```

Comment:
its not clear what the purpose of this is esp. the name may conflict with understanding of Envoy sidecar.

Reply:
it's an http proxy server. this allows me to test whether a pod can reach a service programmatically.

e.g. code<running on mac/circleCI> access services through this HTTPProxy.